doc/md/monitoring/07-notifications.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Notifications
+---
+
+To receive notifications about schema changes in your database, you can set up notifications in Atlas Schema Monitoring.
+Notifications can be sent to Slack, Workplace, or a plain webhook URL.
+
+
+#### Setting up Notifications
+
+To set up notifications, go to the **webhooks** tab of the database you want to monitor and click on the **Add Webhook** button.
```

Comment:
1. Log in to your Atlas Cloud account at https://auth.atlasgo.cloud.
2. Click on "Monitoring" in the top-level navigation on the left side of the screen. 
3. Choose ..